cookbook/tables.md

```diff
@@ -21,7 +21,7 @@ Examples shown in [`Working with tables`](../book/working_with_tables.md) work f
 ───┴───┴───┴───┴───
 ```
 
-Second row in columns `c` and `d` is empty because our `second` table only contained a single row so nushell has nothing to fill the remaining rows with. But what if we wanted the smaller table to 'wrap around' and keep filling the rows? For that we can use the [`group`](../book/commands/group.md) command to split the larger table into subtables, merge each of them with the smaller table and then combine the merged tables together using [`flatten`](../book/commands/flatten.md) command like this:
+Second row in columns `c` and `d` is empty because our `second` table only contained a single row so nushell has nothing to fill the remaining rows with. But what if we wanted the smaller table to 'wrap around' and keep filling the rows? For that we can use the [`group`](../commands/docs/group.md) command to split the larger table into subtables, merge each of them with the smaller table and then combine the merged tables together using [`flatten`](../commands/docs/flatten.md) command like this:
```

Comment:
Why not use absolute path instead? `/commands/docs/flatten.md` instead of `../commands/docs/flatten.md`

Reply:
Just used what was there before, fixed now.